lighthouse-core/scripts/legacy-javascript/summary-signals-nomaps.json

```diff
@@ -372,324 +354,324 @@
       "signals": ""
     },
     {
-      "name": "core-js-3-only-polyfill/es6-array-fill",
-      "signals": "Array.prototype.fill"
+      "name": "core-js-3-only-polyfill/es-array-buffer-constructor",
```

Comment:
The misalignment in the diff is unfortunate, but no need to really review this. All the signals for core-js 3 variants before were simply wrong, this new patch should be consider a new baseline.

Looks like `WeakMap` is used to help define the majority of this polyfils in core js 3, so it's in a lot of variants. for most of the "only-polyfill" variants there is another signal for the polyfill in isolation. Just a few are missing that, ndb.